src/MCPClient/lib/clientScripts/restructureDIPForContentDMUpload.py

```diff
@@ -273,50 +270,22 @@ def getContentdmCollectionFieldInfo(contentdmServer, targetCollection):
     return collectionFieldInfo
 
 
-# Return the dmdSec with the specific ID value. If dublinCore is True, return
-# the <dublincore> child node only.
-def getDmdSec(metsDom, dmdSecId = 'dmdSec_1', dublinCore = True):
-    for node in metsDom.getElementsByTagName('dmdSec'):
-        for k, v in node.attributes.items():
-            if dublinCore and k == 'ID' and v == dmdSecId:
-                # Assumes there is only one dublincore child element.
-                return node.getElementsByTagName('dublincore')[0]
-            else:
-                return node
+def getObjectDirectoryFiles(objectDir):
+    """
+    Get a list of all the files (recursive) in the provided directory.
 
+    "Even though there can be subdirectories in the objects directory,
+    assumes each file should have a unique name." What does this mean?
```

Comment:
I wonder if this means that filenames are (or should be) guaranteed unique across the SIP, regardless of the directory in which they're located? e.g. two subdirectories can't have objects with the same names?